oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/FetchJwksHandlerTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.di.authentication.oidc.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.nimbusds.jose.KeySourceException;
+import org.junit.jupiter.api.Test;
+import uk.gov.di.authentication.oidc.entity.FetchJwksResponse;
+import uk.gov.di.orchestration.shared.services.JwksService;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class FetchJwksHandlerTest {
```

Comment:
I spent a while but couldn't figure out a way of testing the happy path by mocking the response from `jwksService.retrieveJwkFromURLWithKeyId`. I thought it would be OK to cover this case in an integration test once the code to invoke FetchJwksHandler is merged (in a subsequent PR).

Reply:
Update: figured it out